lib/vega_lite/export.ex

```diff
@@ -85,4 +102,96 @@ defmodule VegaLite.Export do
   defp escape_double_quotes(json) do
     String.replace(json, ~s{"}, ~s{\\"})
   end
+
+  @doc """
+  Renders the given graphic as a PNG image and returns
+  its binary content.
+
+  Relies on the `npm` packages mentioned above.
+  """
+  @spec to_png(VegaLite.t()) :: binary()
+  def to_png(vl) do
+    node_convert(vl, "png", "to_png/1")
+  end
+
+  @doc """
+  Renders the given graphic as an SVG image and returns
+  its binary content.
+
+  Relies on the `npm` packages mentioned above.
+  """
+  @spec to_svg(VegaLite.t()) :: binary()
+  def to_svg(vl) do
+    node_convert(vl, "svg", "to_svg/1")
+  end
+
+  @doc """
+  Renders the given graphic into a PDF and returns its
+  binary content.
+
+  Relies on the `npm` packages mentioned above.
+  """
+  @spec to_pdf(VegaLite.t()) :: binary()
+  def to_pdf(vl) do
+    node_convert(vl, "pdf", "to_pdf/1")
+  end
+
+  defp node_convert(vl, format, fn_name) do
+    json = to_json(vl)
+    json_file = System.tmp_dir!() |> Path.join("vl.json")
+    File.write!(json_file, json)
+
+    script_path = find_npm_script!("vl2#{format}", fn_name)
+    {output, 0} = System.cmd(script_path, [json_file])
+
+    File.rm!(json_file)
```

Comment:
```suggestion
    _ = File.rm(json_file)
```

I don't trust File.rm on Windows, sometimes it has delayed file handles, so lets do a best effort, which is fine for a tmp dir.